test/suites/optional/generic-device-plugin.robot

```diff
@@ -113,3 +127,68 @@ Wait For Job Completion And Check Logs
     ...    .items[*].metadata.name
     ${logs}=    Oc Logs    ${pod}    ${NAMESPACE}
     Should Contain    ${logs}    Test successful
+
+Create TtyUSB Devices
+    [Documentation]    Creates dummy ttyUSB devices
+    FOR    ${i}    IN RANGE    ${DEVICE_COUNT}
+        Command Execution    sudo mknod --mode=666 /dev/ttyUSB${i} c 166 ${i}
+    END
+
+Remove Dummy TtyUSB Devices
+    [Documentation]    Removes dummy ttyUSB devices
+    FOR    ${i}    IN RANGE    ${DEVICE_COUNT}
+        Command Execution    sudo rm -f /dev/ttyUSB${i}
+    END
+
+TtyUSB Glob Test Setup
+    [Documentation]    Setup for ttyUSB glob pattern test - creates devices, configures GDP, waits for allocation
+    Create TtyUSB Devices
+    Drop In MicroShift Config    ${GDP_CONFIG_SERIAL_GLOB}    10-gdp
+    Restart MicroShift
+    Wait Until Device Is Allocatable    ${DEVICE_COUNT}
+
+TtyUSB Glob Test Teardown
+    [Documentation]    Cleanup for ttyUSB glob pattern test
+    Oc Delete    pod --all -n ${NAMESPACE}
+    Remove Dummy TtyUSB Devices
+    Disable GDP
```

Comment:
This keyword also restarts microshift - since every test in this suite is setting dropin and restarting MicroShift, we could just remove the dropin.

In other words go from:
> - setup test: set up dropin, restart ushift
> - run test
> - teardown: remove dropin, restart ushift
> - setup test: set up dropin, restart ushift
> ...

we could do
> - setup test: set up dropin, restart ushift
> - run test
> - teardown: remove dropin
> - setup test: set up dropin, restart ushift
> ...